_includes/releases/v22.1/v22.1.19.md

```diff
@@ -0,0 +1,62 @@
+
+- Queries with invalid syntax are now logged at the `INFO` level on the [`SQL_EXEC` logging channel](../v22.1/sql-audit-logging.html). Previously, they were logged at the `ERROR` level. [#101090][#101090]
+
+<h3 id="v22-1-19-sql-language-changes">SQL language changes</h3>
+
+- Added the `prepared_statements_cache_size` [session setting](../v22.1/set-vars.html) which helps to prevent [prepared statement](../v22.1/savepoint.html#savepoints-and-prepared-statements) leaks by automatically deallocating the least-recently-used prepared statements when the cache reaches a given size. [#99264][#99264]
+
+<h3 id="v22-1-19-db-console-changes">DB Console changes</h3>
+
+- New data is now auto-fetched every 5 mins on the [**Statement and Transaction Fingerprints**](../v22.1/ui-statements-page.html#statement-fingerprint-page) pages. [#100702][#100702]
+
+<h3 id="v22-1-19-bug-fixes">Bug fixes</h3>
+
+- Previously, [`ADD COLUMN ... DEFAULT cluster_logical_timestamp()`](../v22.1/alter-table.html) would crash the node and leave the table in a corrupt state. The root cause is a `nil` pointer dereference. The bug is now fixed by returning an unimplemented error and hence disallowing using the [builtin function](../v22.1/functions-and-operators.html#array-functions) as the default value when backfilling. [#99682][#99682]
+- Fixed a bug where the stats columns on the [**Transaction Fingerprint Overview**](../v22.1/ui-transactions-page.html#transaction-details-page) page was continuously incrementing. [#99405][#99405]
+- Fixed a bug that prevented the garbage-collection job for the [`TRUNCATE TABLE`](../v22.1/truncate.html) command from successfully finishing if the table descriptor had already been garbage-collected. The garbage-collection job now succeeds in this situation by handling the missing descriptor edge case. [#100146][#100146]
+- Fixed a bug present in v21.1 that would cause the SQL gateway node to crash if a [view was created](../v22.1/create-view.html) with circular or self-referencing dependencies. [#100165][#100165]
+- Fixed a bug in evaluation of `ANY`, `SOME`, and `ALL` [sub-operators](../v22.1/functions-and-operators.html#operators) that would cause expressions like `NULL = ANY(ARRAY[]::INT[])` to return `NULL` instead of `FALSE`. [#100363][#100363]
+- Fixed a bug that could prevent a cached query with a [user-defined type](../v22.1/create-type.html) reference from being invalidated even after a schema change that should prevent the type from being resolved. [#100358][#100358]
+- Fixed a bug existing before v22.1 that could cause a projected expression to replace column references with incorrect values. [#100368][#100368]
+- Fixed a bug where the physical disk space of some tables could not be calculated. [#100937][#100937]
+- Fixed a bug such that the `crdb_internal.deserialize_session` function works properly with prepared statements that have more param type hints than params. [#101363][#101363]
```

Comment:
Nits: "Fixed a bug so that..."

Perhaps link: crdb-internal.html

"param(s)" --> "parameter(s)"